apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -251,7 +251,7 @@ provides a simple method to accomplish this:
 
 If you want to diagnose session management issues,
 `SpannerConnection` provides a method to fetch a snapshot of
-the statistics for the session pool associated with the database
+the statistics for the session pool associated with the SessionPoolKey
 associated with the connection:
 
-{{sample:SpannerConnection.GetSessionPoolDatabaseStatistics}}
+{{sample:SpannerConnection.GetSessionPoolKeyStatistics}}
```

Comment:
```suggestion
{{sample:SpannerConnection.GetSessionPoolStatistics}}
```

Reply:
Additionally, it's worth building the docs to check this:

- From the top level:
- `./build.sh Google.Cloud.Spanner.Data`
- `cd docs`
- `./builddocs.sh Google.Cloud.Spanner.Data`
- `../packages/docfx.2.59.4/docfx serve output/Google.Cloud.Spanner.Data/site`
- Open `localhost:8080` and check the docs look okay